ui/pages/sample/components/sample-network-display.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { AlignItems, Display } from '../../../helpers/constants/design-system';
+import {
+  Text,
+  AvatarNetwork,
+  AvatarNetworkSize,
+  Box,
+} from '../../../components/component-library';
+
+type SampleNetworkDisplayProps = {
```

Comment:
These are very few types so we can use it in the same file but with a lot of types, it's better to create a separate constants file